src/explorer/src/components/Endpoint/Geocoding/StreetZone/meta.js

```diff
@@ -1,61 +1,32 @@
-const initialState = {
-  street: '',
-  zone: '',
-  spatialReference: '26912',
-  acceptScore: '70',
-  pobox: false,
-  locators: 'all',
-  format: 'default',
-  suggest: '0',
-  scoreDifference: false,
-  callback: ''
-};
+import { object, string, number, boolean, addMethod, mixed } from 'yup';
 
-const defaultAttributes = {
-  street: {
-    placeholder: '123 south main street',
-    type: 'string',
-    required: true
-  },
-  zone: {
-    placeholder: 'SLC or 84111',
-    type: 'string',
-    required: true
-  },
-  sr: {
-    placeholder: 26912,
-    type: 'int',
-    required: false
-  },
-  acceptScore: {
-    placeholder: 70,
-    type: 'int',
-    required: false
-  },
-  pobox: {
-    type: 'boolean',
-    required: false
-  },
-  locators: {
-    placeholder: 'all',
-    type: 'string',
-    required: false
-  },
-  suggest: {
-    placeholder: 0,
-    type: 'int',
-    required: false
-  },
-  scoreDifference: {
-    placeholder: 0,
-    type: 'int',
-    required: false
-  },
-  callback: {
-    placeholder: 'callback',
-    type: 'string',
-    required: false
-  }
-};
+const schema = object().shape({
+  street: string().matches(/^\d*\s\D/, {
+    message: 'Must be valid street address',
+    excludeEmptyString: true
+  }).required().meta({ placeholder: '123 south main street'}),
+  zone: string().matches(/(^84\d{3}$|^\D)/, {
+      message: 'Must be valid Utah zip code or city name',
+      excludeEmptyString: true
+    }).required().meta({ placeholder: 'SLC or 84111' }),
```

Comment:
**nitpick**: the default yup schema messages are lowercase following a `this is` or `this must` template. I think we should match that for consistency. 
```suggestion
  street: string().matches(/^\d*\s\D/, {
    message: 'this must be a valid street address',
    excludeEmptyString: true
  }).required().meta({ placeholder: '123 south main street'}),
  zone: string().matches(/(^84\d{3}$|^\D)/, {
      message: 'this must be a valid Utah zip code or city name',
      excludeEmptyString: true
    }).required().meta({ placeholder: 'SLC or 84111' }),
```